src/Compiler/SyntaxTree/SyntaxTreeOps.fs

```diff
@@ -468,7 +468,7 @@ let mkSynDotMissing mDot m l =
     | SynExpr.LongIdent (isOpt, SynLongIdent (lid, dots, trivia), None, _) ->
         // REVIEW: MEMORY PERFORMANCE: This list operation is memory intensive (we create a lot of these list nodes)
         SynExpr.LongIdent(isOpt, SynLongIdent(lid, dots @ [ mDot ], trivia), None, m)
-    | SynExpr.Ident id -> SynExpr.LongIdent(false, SynLongIdent([ id ], [ mDot ], []), None, m)
+    | SynExpr.Ident id -> SynExpr.LongIdent(false, SynLongIdent([ id ], [ mDot ], [ None ]), None, m)
```

Comment:
@dsyme could we have this fix as well, please?

Reply:
@nojaf so this is all ready?

Reply:
Yup, this is the most significant fix.
And if we did miss another case, `IdentsWithTrivia` will not blow up if a trivia is missing.